code/src/App.js

```diff
@@ -1,9 +1,19 @@
-import React from 'react'
+import React from 'react';
+
+import Wrappers from './components/Wrapper';
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+      <div>
```

Comment:
Good to see the react fragments being used here. I am only curious to know why you have added div tags also?